### PR TITLE
PR guidance: Fork or delete branches after merge

### PR DESCRIPTION
**Pull request by dutchshark**
Addressing issue #860

### CODE REVIEW

File: governance/github.md [hunk 1]
```diff
@@ -38,6 +38,10 @@ The "master" branch is "protected" (even for admins), with these requirements:
...
 
+## Housekeeping
+To maintain the main branch in a feasible way the Pull Requests shall come from forked repositories from tag-security. After which an unique name should be assigned to identify what will be changed in the forked repository by utilizing branches. 
 
-
-
+After being merged the branch will be deleted.
+Next contributions shall be in another fresh branch. 
+This way we keep the repository clean and allows a faster acceptance as it's clear what exactly is addressed.
+This highly limits the amount of branches and stops having branches around for longer then needed.
```
Comment (PushkarJ) [inline governance/github.md:47]:
```suggestion
## Housekeeping
To maintain the main branch in a feasible way the Pull Requests shall come
from forked repositories from tag-security. After which an unique name
should be assigned to identify what will be changed in the forked repository
by utilizing branches. 

After being merged the branch will be deleted.
Next contributions shall be in another fresh branch. 
This way we keep the repository clean and allows a faster acceptance as it's
clear what exactly is addressed.

This highly limits the amount of branches and stops having branches around
for longer then needed.
```

**Review by PushkarJ — CHANGES_REQUESTED** [review]

File: governance/github.md (hunk 1 above)
Comment (PushkarJ) [reply governance/github.md:47]: @dutchshark if you commit this suggestion, I believe it should allow us to pass the CI lint job

**Comment by ultrasaurus** [comment]
From a process perspective this seems like a good change!

File: governance/github.md (hunk 1 above)
Comment (PushkarJ) [reply governance/github.md:47]: Committed this suggestion as the original author has been unavailable

**Review by PushkarJ — APPROVED** [review]

**Comment by PushkarJ** [comment]
Lint errors should be fixed once this is merged: https://github.com/cncf/tag-security/pull/964 

We should update the branch to let the CI run again

File: governance/github.md [hunk 2]
```diff
@@ -41,3 +41,15 @@ requirements:
   - must be up to date with master
   - title must not indicate work in
     progress ([WIP](https://github.com/apps/wip))
+
+## Housekeeping
+To maintain the main branch in a feasible way the Pull Requests shall come
+from forked repositories from tag-security. After which an unique name
+should be assigned to identify what will be changed in the forked
+repository by utilizing branches.
```
Comment (PushkarJ) [inline governance/github.md:49]:
```suggestion
repository by utilizing branches.
```

File: governance/github.md [hunk 3]
```diff
@@ -41,3 +41,15 @@ requirements:
...
     progress ([WIP](https://github.com/apps/wip))
+
+## Housekeeping
+To maintain the main branch in a feasible way the Pull Requests shall come
+from forked repositories from tag-security. After which an unique name
+should be assigned to identify what will be changed in the forked
+repository by utilizing branches. 
+
+After being merged the branch will be deleted.
+Next contributions shall be in another fresh branch.
```
Comment (PushkarJ) [inline governance/github.md:52]:
```suggestion
Next contributions shall be in another fresh branch.
```

File: governance/github.md [hunk 4]
```diff
@@ -41,3 +41,15 @@ requirements:
   - must be up to date with master
   - title must not indicate work in
     progress ([WIP](https://github.com/apps/wip))
+
+## Housekeeping
```
Comment (PushkarJ) [inline governance/github.md:45]:
```suggestion
## Housekeeping

```

**Pull request merged**